tests/profiler/bob_user_workflow_verbose_profiler_config.yml

```diff
@@ -0,0 +1,216 @@
+          data_connector_query:
+            batch_filter_parameters:
+              year: 2020 # TODO: Can this be relative to the current year? (e.g. specify last year vs the exact year)
+        metric_configuration:
+          metric_name: column.mean # per batch metric
+          # Domain kwargs here refer to the current domain being processed - the list of domains coming from the DomainBuilder (here the list of columns filtered by SimpleColumnSuffixDomainBuilder)
+          metric_domain_kwargs: $domain.domain_kwargs
+        aggregate_fn: mean # Not necessarily a metric, but some function to apply to all of the batches - can be different from metric_name
+      - name: my_stdev_of_means_parameter_for_batches_in_previous_full_year
+        # Parameter computes the stdev of means of all batches in the previous full year
+        class_name: MultiBatchAggregateFunctionParameterBuilder
+        module_name: great_expectations.profiler.parameter_builder.multi_batch_metric_parameter_builder
+        batch_request:
+          data_connector_query:
+            batch_filter_parameters:
+              year: 2020
+        metric_configuration:
+          metric_name: column.mean # per batch metric
+          metric_domain_kwargs: $domain.domain_kwargs
+        aggregate_fn: stdev # note different than metric computed in metric_configuration
+    expectation_configuration_builders:
+      - expectation_type: expect_column_values_to_be_between
+        class_name: DefaultExpectationConfigurationBuilder
+        module_name: great_expectations.profiler.expectation_configuration_builder.default_expectation_configuration_builder
+        # Note: literals are and will continue to be supported (you do not have to add to "variables" first, unless you want to reference in multiple locations):
+        min_value: 0
+        max_value: $variables.max_amount
+        column: $domain.domain_kwargs.column
+      - expectation_type: expect_column_mean_to_be_between
+        class_name: DefaultExpectationConfigurationBuilder
+        module_name: great_expectations.profiler.expectation_configuration_builder.default_expectation_configuration_builder
+        column: $domain.domain_kwargs.column
+        # Note: proposal to reference here in dot notation by the aggregate_fn name
+        min_value: $parameter.my_mean_parameter_for_batches_in_previous_full_year.mean * 0.8
+        max_value: $parameter.my_mean_parameter_for_batches_in_previous_full_year.mean * 1.2
+        # Alternatively, we could have used a MultiBatchParameterBuilder and processed the means in the ExpectationConfigurationBuilder as in my_rule_for_ids
+        # min_value: mean($parameter.my_mean_parameter_for_batches_in_previous_full_year.means) * 0.8
+        # max_value: mean($parameter.my_mean_parameter_for_batches_in_previous_full_year.means) * 1.2
+        # Note: this would need us to add `mean()` to our GE language parsing these evaluations
```

Comment:
Ah. I asked about this above - thanks for clarifying here! I think we could potentially wind up with many ways of accessing the same parameters, and I would question whether we need all of these.

Reply:
Again, as mentioned before, I would prefer to implement the `Mean` in an aggregator class first, before promoting it to the configuration syntax level.